trim_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class TrimDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.strip
```

Comment:
- [ ] We should trim the name down to 10 characters, instead of just removing whitespace.

  ```
  @nameable.correct_name[0..9]
  ```